File: Python/Primer/files.py
# create file
with open('test.txt', 'w') as file:
    pass

# read from file
with open('test.txt', 'r') as file:
    file.read()

# or
content = open('test.txt', 'r').read()

# read lines
with open('test.txt', 'r') as file:
    for line in file.readlines():
        # do something
        pass

# write to file
with open('test.txt', 'w') as file:
    file.write('This is some text to write.')

File: Pascal/Primer/files.pas
// 2020-02
program Files;

var
    // variable for file
    f: file of string;
    // variable for text
    text: string;
begin
    // write to file
    assign(f, '_file.txt');
    rewrite(f);
    write(f, 'hello pascal');
    close(f);

    // read from file
    assign(f, '_file.txt');
    reset(f);
    while not eof(f) do
        begin
            read(f, text);
            writeln(text);
        end;
    // end while
    close(f);
    // hello pascal
end.